circus/util.py

```diff
@@ -693,11 +693,14 @@ def configure_logger(logger, level='INFO', output="-", loggerconfig=None,
         elif output.startswith('syslog://'):
             # URLs are syslog://host[:port]?facility or syslog:///path?facility
             info = urlparse(output)
-            facility = 'user'
+
+            # find out the specified facility
             if info.query in logging.handlers.SysLogHandler.facility_names:
                 facility = info.query
+            else:
+                facility = 'user'
             if info.netloc:
-                address = (info.netloc, info.port or 514)
+                address = (info.hostname, info.port or 514)
```

Comment:
Could you please add a test exposing this bug? 

It can be as simple as:

```
def test_syslog_output_format(self):
    # if syslog host parsing error occurs this test will fail
    configure_logger(None, output='syslog://localhost:514?test')
```

And you can put it either in `test_circusd` or `test_logging` or `test_config`, wherever you fill comfortable.